bindings/jupyroot/CMakeLists.txt

```diff
@@ -41,13 +41,10 @@ foreach(val RANGE ${how_many_pythons})
 
   set(libname JupyROOT${python_under_version_string})
 
-  add_library(${libname} SHARED src/IOHandler.cxx)
-
   # libJupyROOT uses ROOT headers from source dirs and depends on Core
+  ROOT_LINKER_LIBRARY(${libname} src/IOHandler.cxx DEPENDENCIES ${python_library} Core ${CMAKE_DL_LIBS} CMAKENOEXPORT)
```

Comment:
@oshadura should we specify `${python_library}` as `LIBRARIES` and not `DEPENDENCIES`?

Reply:
Good catch!